### ISSUE TITLE
Feature - Search for particular currency

### ISSUE THREAD

**Issue by Otagera** [LaniJ/Invoice-Receipt-Generator#7]
So When I wanted to select a currency I noticed that I had to scroll down to locate the particular currency I wanted to select
<img width="1175" alt="Screenshot 2023-07-28 at 09 03 39" src="https://github.com/LaniJ/invoice-dragon/assets/32098712/d6b5e960-af18-413a-9be9-6cf97bb4b59e">

So I would like a feature that gives me the option to search for a particular currency.

**Comment by lumie31** [comment]
Is this already in development? It would be nice to add a label e.g. `in progress` so that someone else doesn't work on what's already in progress

**Comment by Otagera** [comment]
Yeah, at attempt was made [here](https://github.com/LaniJ/invoice-dragon/pull/8)
